--- INSTALL/tstiee.f    2021-01-05 11:24:57.340644843 -0500
+++ INSTALL/tstiee.f    2021-01-05 11:26:12.284656123 -0500
@@ -281,76 +281,76 @@
       IF( C2.EQ.'GE' ) THEN
          IF( C3.EQ.'TRF' ) THEN
             IF( SNAME ) THEN
-               NB = 64
+               NB = 512
             ELSE
-               NB = 64
+               NB = 512
             END IF
          ELSE IF( C3.EQ.'QRF' .OR. C3.EQ.'RQF' .OR. C3.EQ.'LQF' .OR.
      $            C3.EQ.'QLF' ) THEN
             IF( SNAME ) THEN
-               NB = 32
+               NB = 512
             ELSE
-               NB = 32
+               NB = 512
             END IF
          ELSE IF( C3.EQ.'HRD' ) THEN
             IF( SNAME ) THEN
-               NB = 32
+               NB = 512
             ELSE
-               NB = 32
+               NB = 512
             END IF
          ELSE IF( C3.EQ.'BRD' ) THEN
             IF( SNAME ) THEN
-               NB = 32
+               NB = 512
             ELSE
-               NB = 32
+               NB = 512
             END IF
          ELSE IF( C3.EQ.'TRI' ) THEN
             IF( SNAME ) THEN
-               NB = 64
+               NB = 512
             ELSE
-               NB = 64
+               NB = 512
             END IF
          END IF
       ELSE IF( C2.EQ.'PO' ) THEN
          IF( C3.EQ.'TRF' ) THEN
             IF( SNAME ) THEN
-               NB = 64
+               NB = 512
             ELSE
-               NB = 64
+               NB = 512
             END IF
          END IF
       ELSE IF( C2.EQ.'SY' ) THEN
          IF( C3.EQ.'TRF' ) THEN
             IF( SNAME ) THEN
-               NB = 64
+               NB = 512
             ELSE
-               NB = 64
+               NB = 512
             END IF
          ELSE IF( SNAME .AND. C3.EQ.'TRD' ) THEN
-            NB = 32
+            NB = 512
          ELSE IF( SNAME .AND. C3.EQ.'GST' ) THEN
-            NB = 64
+            NB = 512
          END IF
       ELSE IF( CNAME .AND. C2.EQ.'HE' ) THEN
          IF( C3.EQ.'TRF' ) THEN
-            NB = 64
+            NB = 512
          ELSE IF( C3.EQ.'TRD' ) THEN
-            NB = 32
+            NB = 512
          ELSE IF( C3.EQ.'GST' ) THEN
-            NB = 64
+            NB = 512
          END IF
       ELSE IF( SNAME .AND. C2.EQ.'OR' ) THEN
          IF( C3( 1:1 ).EQ.'G' ) THEN
             IF( C4.EQ.'QR' .OR. C4.EQ.'RQ' .OR. C4.EQ.'LQ' .OR.
      $          C4.EQ.'QL' .OR. C4.EQ.'HR' .OR. C4.EQ.'TR' .OR.
      $          C4.EQ.'BR' ) THEN
-               NB = 32
+               NB = 512
             END IF
          ELSE IF( C3( 1:1 ).EQ.'M' ) THEN
             IF( C4.EQ.'QR' .OR. C4.EQ.'RQ' .OR. C4.EQ.'LQ' .OR.
      $          C4.EQ.'QL' .OR. C4.EQ.'HR' .OR. C4.EQ.'TR' .OR.
      $          C4.EQ.'BR' ) THEN
-               NB = 32
+               NB = 512
             END IF
          END IF
       ELSE IF( CNAME .AND. C2.EQ.'UN' ) THEN
@@ -358,13 +358,13 @@
             IF( C4.EQ.'QR' .OR. C4.EQ.'RQ' .OR. C4.EQ.'LQ' .OR.
      $          C4.EQ.'QL' .OR. C4.EQ.'HR' .OR. C4.EQ.'TR' .OR.
      $          C4.EQ.'BR' ) THEN
-               NB = 32
+               NB = 512
             END IF
          ELSE IF( C3( 1:1 ).EQ.'M' ) THEN
             IF( C4.EQ.'QR' .OR. C4.EQ.'RQ' .OR. C4.EQ.'LQ' .OR.
      $          C4.EQ.'QL' .OR. C4.EQ.'HR' .OR. C4.EQ.'TR' .OR.
      $          C4.EQ.'BR' ) THEN
-               NB = 32
+               NB = 512
             END IF
          END IF
       ELSE IF( C2.EQ.'GB' ) THEN
@@ -373,13 +373,13 @@
                IF( N4.LE.64 ) THEN
                   NB = 1
                ELSE
-                  NB = 32
+                  NB = 512
                END IF
             ELSE
                IF( N4.LE.64 ) THEN
                   NB = 1
                ELSE
-                  NB = 32
+                  NB = 512
                END IF
             END IF
          END IF
@@ -389,30 +389,30 @@
                IF( N2.LE.64 ) THEN
                   NB = 1
                ELSE
-                  NB = 32
+                  NB = 512
                END IF
             ELSE
                IF( N2.LE.64 ) THEN
                   NB = 1
                ELSE
-                  NB = 32
+                  NB = 512
                END IF
             END IF
          END IF
       ELSE IF( C2.EQ.'TR' ) THEN
          IF( C3.EQ.'TRI' ) THEN
             IF( SNAME ) THEN
-               NB = 64
+               NB = 512
             ELSE
-               NB = 64
+               NB = 512
             END IF
          END IF
       ELSE IF( C2.EQ.'LA' ) THEN
          IF( C3.EQ.'UUM' ) THEN
             IF( SNAME ) THEN
-               NB = 64
+               NB = 512
             ELSE
-               NB = 64
+               NB = 512
             END IF
          END IF
       ELSE IF( SNAME .AND. C2.EQ.'ST' ) THEN
