diff --git a/pk/syscall.c b/pk/syscall.c
index b47c237..aa997a4 100644
--- a/pk/syscall.c
+++ b/pk/syscall.c
@@ -607,6 +607,7 @@ long do_syscall(long a0, long a1, long a2, long a3, long a4, long a5, unsigned l
     [SYS_brk] = sys_brk,
     [SYS_uname] = sys_uname,
     [SYS_getpid] = sys_getpid,
+    [SYS_gettid] = sys_getpid,
     [SYS_getuid] = sys_getuid,
     [SYS_geteuid] = sys_getuid,
     [SYS_getgid] = sys_getuid,
@@ -638,6 +639,7 @@ long do_syscall(long a0, long a1, long a2, long a3, long a4, long a5, unsigned l
     [SYS_chdir] = sys_chdir,
     [SYS_set_tid_address] = sys_stub_nosys,
     [SYS_set_robust_list] = sys_stub_nosys,
+    [SYS_futex] = sys_stub_success,
     [SYS_madvise] = sys_stub_nosys,
   };
 
diff --git a/pk/syscall.h b/pk/syscall.h
index db613ac..19a2ffe 100644
--- a/pk/syscall.h
+++ b/pk/syscall.h
@@ -57,6 +57,8 @@
 #define SYS_set_robust_list 99
 #define SYS_madvise 233
 #define SYS_statx 291
+#define SYS_futex 98
+#define SYS_gettid 178
 
 #define OLD_SYSCALL_THRESHOLD 1024
 #define SYS_open 1024
